common/unified/components/format_conversion_kernels.cpp

```diff
@@ -21,16 +24,23 @@ void convert_ptrs_to_idxs(std::shared_ptr<const DefaultExecutor> exec,
                           const RowPtrType* ptrs, size_type num_blocks,
                           IndexType* idxs)
 {
+    const auto num_elements = exec->copy_val_to_host(ptrs + num_blocks);
```

Comment:
maybe we just paste the idxs size from outside. We should have it already when allocating the idxs array